ledger/tracker.go

```diff
@@ -296,6 +300,9 @@ func (tr *trackerRegistry) initialize(l ledgerForTracker, trackers []ledgerTrack
 			tr.txtail = txtail
 		}
 	}
+
+	tr.alwaysCommit = alwaysCommit
```

Comment:
looks unused and set to false everywhere. Remove?

Reply:
I do not think this flag should exist at all since it is only used for testing, and the only place where it is needed is `TestFirstStageInfoPruning`. I suggest to do the following:
instead of 
```
ml.trackers.newBlock(blk, delta)
ml.trackers.committedUpTo(i)
```
call 
```
ml.trackers.lastFlushTime = time.Time{}
ml.trackers.newBlock(blk, delta)
```

Reply:
done